File: Javascript/8-Kyu/Remove-exclamation-marks.js
function removeExclamationMarks(s) {

    return s.split('!').join('');

};

// Tests
console.log(removeExclamationMarks('Hello World!'));  // --> 'Hello World'
console.log(removeExclamationMarks('!!Hey!!!'));  // --> 'Hey'
console.log(removeExclamationMarks('H!e!l!l!o!'));  // --> 'Hello'

File: Python/8-Kyu/Remove-exclamation-marks.py
# Remove exclamation marks
def remove_exclamation_marks(s):
    
    return s.replace('!', '')

# Tests
print(remove_exclamation_marks("Hello World!"))  # --> "Hello World"
print(remove_exclamation_marks("Hello World!!!"))  # --> "Hello World"
print(remove_exclamation_marks("Hi! Hello!"))  # --> "Hi Hello"
print(remove_exclamation_marks(""))  # --> ""
print(remove_exclamation_marks("Oh, no!!!"))  # --> "Oh, no"
